.github/workflows/destroy.yaml

```diff
@@ -133,6 +133,9 @@ jobs:
           ARM_SUBSCRIPTION_ID: ${{ secrets.ARM_SUBSCRIPTION_ID }}
           ARM_TENANT_ID: ${{ secrets.ARM_TENANT_ID }}
 
+          # Passing dummy variables to terraform destroy, because destroy needs input variables to be defined, but uses the state.
+          TF_VAR_application_sp_client_secret: dummy
```

Comment:
do we need this as this is already in .tfvars ?Asking as we do not need such dummy variable for anything else.

Reply:
It is not in the .tfvars, as it contains sensitive data I think. Same pattern is applied for participants in line 85.

Reply:
Right, I see it is secret.